extensions/batch/chathistory.md

```diff
@@ -0,0 +1,41 @@
+---
+title: IRCv3 CHATHISTORY Batch Type
+layout: spec
+copyrights:
+  -
+    name: "Peter Powell"
+    period: "2015"
+    email: "petpow@saberuk.com"
+---
+## CHATHISTORY Batch Type
+
+This document describes the format of the CHATHISTORY batch type.
+
+This batch type requires the [`server-time` capability][server-time] to be
+enabled by the client. If that capability has not been enabled by the client
+then this batch type MUST NOT be sent by the server.
+
+The CHATHISTORY batch type takes one parameter. This parameter contains the
+target of messages within the batch. The target MUST either be the nick of the
+receiving client for private messages or the name of a channel which the
+receiving client is in for public messages.
+
+When a client performs an action which requires chat history to be relayed to
+it (e.g. joining a channel with InspIRCd's chanhistory mode enabled) the server
+MUST put all PRIVMSGs and NOTICEs into a single CHATHISTORY batch. All
+messages MUST be [tagged][server-time] with the time at which they were
+originally sent.
+
+Clients which have not implemented support for the CHATHISTORY batch type
+SHOULD treat the PRIVMSGs and NOTICEs within the batch using their standard
+message handling method.
```

Comment:
One possibility is to remove this paragraph entirely, another one is to change it to be a reminder of the above quote.